### PR TITLE
[DRAFT] Make the designer page keyboard accessible

### PR DESCRIPTION
**Pull request by PankajBhojwani**
WIP of our branch where we are working on updating the designer page to be keyboard accessible

Work items:
- [x] focus should be able to land on a peer
- [x] remove control should be keyboard activatable
- [ ] drag control should be keyboard activatable
- [x] bind control should be keyboard activatable
- [x] if the peer is wrapping an action, the action should still be invokable
- [x] we need some way to communicate the keybindings to the user

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7159)

### CODE REVIEW

**Comment by carlos-zamora** [comment]
I tried adding the ability to move the peer. But out of nowhere, my project didn't want to build anymore. Progress can be found in dev/cazamor/designer_page/fork

File: source/nodejs/adaptivecards-designer/src/card-designer-surface.ts [hunk 1]
```diff
@@ -575,6 +584,33 @@ export class CardDesignerSurface {
...
+                    for (let i = 0; i < buttons.length; i++) {
+                        let button = buttons.item(i);
+                        if (button instanceof HTMLButtonElement) {
+                            if (button.title == "Add an action") {
+                                button.click();
+                            }
+                        }
+                    }
+                }
+            }
```
Comment (paulcam206) [inline source/nodejs/adaptivecards-designer/src/card-designer-surface.ts:611]: Do you really intend to invoke _every_ button that matches here?

If **"No"**, 

```suggestion
if (this._selectedPeer && e.ctrlKey) {
    const keyToText = {
        "b": "Bind...",
        "x": "Add an action"
    };

    if (keyToText[e.key]) {
        Array.from(this._peerCommandsHostElement.children).find(e => e instanceof HTMLButtonElement && e.innerText == keyToText[e.key]);
    }
}
```

(if **"Yes"**, check the other comment)

Comment (paulcam206) [inline source/nodejs/adaptivecards-designer/src/card-designer-surface.ts:611]: If **"Yes"**:

```suggestion
if (this._selectedPeer && e.ctrlKey) {
    const keyToText = {
        "b": "Bind...",
        "x": "Add an action"
    };

    if (keyToText[e.key]) {
        Array.from(this._peerCommandsHostElement.children).filter(e => e instanceof HTMLButtonElement && e.innerText == keyToText[e.key]).forEach(e => e.click());
    }
}
```

**Review by paulcam206 — APPROVED** [review]
some simplification possible, but looks good

**Comment by PankajBhojwani** [comment]
Closing this because it turns out the designer page requires a larger overhaul. Not deleting the branch though in case this gets worked on at some point.

**Pull request closed without merging**